toml/parser.ts

```diff
@@ -0,0 +1,312 @@
+// Copyright 2018-2019 the Deno authors. All rights reserved. MIT license.
+import { existsSync } from "../fs/exists.ts";
+
+const deepAssign = (target: object, ...sources: object[]): object => {
```

Comment:
Can you move this to `//util/deep_assign.ts`

Also https://deno.land/style_guide.html#toplevelfunctionsshouldnotusearrowsyntax

Reply:
was a draft, tried to make it work. Also will add tests for it as we will expose it direclty in std.